tests/cephfs/cephfs_scale/cephfs_scale_utils.py

```diff
@@ -0,0 +1,548 @@
+"""
+This is cephfs Scale  utility module
+It contains all the re-useable functions related to cephfs scale tests
+
+"""
+
+import gzip
+import json
+import os
+import random
+import re
+import shutil
+import string
+import threading
+from concurrent.futures import ThreadPoolExecutor, as_completed
+from datetime import datetime
+
+from ceph.ceph import CommandFailed
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from utility.log import Log
+
+log = Log(__name__)
+
+logging_thread = None
+mds_logging_thread = None
+stop_event = threading.Event()
+
+results = {}
+
+
+class CephfsScaleUtils(object):
+    def __init__(self, ceph_cluster):
+        """
+        CephFS Scale Utility object
+        Args:
+            ceph_cluster (ceph.ceph.Ceph): ceph cluster
+        """
+
+        self.mons = ceph_cluster.get_ceph_objects("mon")
+        self.mgrs = ceph_cluster.get_ceph_objects("mgr")
+        self._mdss = ceph_cluster.get_ceph_objects("mds")
+        self.osds = ceph_cluster.get_ceph_objects("osd")
+        self.clients = ceph_cluster.get_ceph_objects("client")
+        self.fs_util = FsUtils(ceph_cluster)
+
+    def mount_and_create_subvolumes(
+        self,
+        fs_util_v1,
+        clients,
+        subvolume_count,
+        default_fs,
+        subvolume_name,
+        subvolume_group_name,
+        mount_type,
+    ):
+        """
+        Create subvolumes and mount them using the specified mount type.
+
+        :param fs_util_v1: FsUtilsV1 object
+        :param clients: List of client objects
+        :param subvolume_count: Number of subvolumes to create
+        :param default_fs: Default file system name
+        :param subvolume_name: Base name for subvolumes
+        :param subvolume_group_name: Name of the subvolume group
+        :param mount_type: Type of mount to use ("fuse" or "kernel")
```

Comment:
can we have coverage for nfs too?

Reply:
NFS QE Team does NFS coverage, so I skipped adding NFS.